### ISSUE TITLE
Ganesha 3.3 Debian Buster missing systemd file ?

### ISSUE THREAD

**Issue by alaunay** [nfs-ganesha/nfs-ganesha-debian#35]
Hello,

I tried upgrading our nfs-ganesha to the latest 3.3 version on a debian buster... And it seems I've a little trouble:

`ii  nfs-ganesha                          3.3-2                        amd64        nfs-ganesha is a NFS server in User Space`

(taken from https://download.nfs-ganesha.org/3/LATEST/Debian/buster/amd64/apt via apt)

```
# systemctl start nfs-ganesha
Failed to start nfs-ganesha.service: Unit nfs-ganesha.service not found.

# dpkg -L nfs-ganesha | grep '/lib/systemd/system'
/lib/systemd/system
/lib/systemd/system/nfs-ganesha-config.service
/lib/systemd/system/nfs-ganesha-lock.service
```

Am I missing something, or is the file missing from the package ?

**Comment by kalebskeithley** [comment]
fixed in nfs-ganesha-3.3-3. Thanks for bringing this to our attention.

**Comment by alaunay** [comment]
I confirm it works fine. Thanks !
